app/controllers/api/v0/serialisers.rb

```diff
@@ -101,19 +101,27 @@ def address_block(office, include_local_authority:)
 
       def opening_time_block(office, time_type)
         days = %w[Monday Tuesday Wednesday Thursday Friday Saturday Sunday]
-        open_days = days.reject { |day| office["#{time_type}_hours_#{day.downcase}"].nil? }
+        case time_type
+        when :office
+          opening_times = office.opening_hours
+          notes = office.opening_hours_information
+        when :telephone
+          opening_times = office.telephone_advice_hours
+          notes = office.telephone_advice_hours_information
+        end
+        open_days = days.reject { |day| opening_times[day.downcase.to_sym].nil? }
```

Comment:
I'm wondering if negating this would make more sense here, i.e.

```ruby
open_days = days.select { |day| opening_times[day.downcase_to_sym].present? }
```